src/main/java/uk/gov/hmcts/divorce/divorcecase/model/RetiredFields.java

```diff
@@ -91,4 +98,10 @@ public static int getVersion() {
     private static TriConsumer<Map<String, Object>, String, Object> moveTo(String newFieldName) {
         return (data, key, val) -> data.put(newFieldName, val);
     }
+
+    private static boolean shouldMigrateSOT(Map<String, Object> data) {
+        return Boolean.parseBoolean(System.getenv().get("SOT_MIGRATION_ENABLED"))
```

Comment:
better to use ` Boolean.parseBoolean(System.getenv().getOrDefault("SOT_MIGRATION_ENABLED","false"))`

Reply:
Done